src/components/scenes/Fio/FioStakingChangeScene.tsx

```diff
@@ -92,7 +92,7 @@ export const FioStakingChangeScene = withWallet((props: Props) => {
     }
   }
 
-  const onAmountChanged = (nativeAmount: string, exchangeAmount: string) => {
+  const onAmountsChanged = ({ exchangeAmount, nativeAmount }: { nativeAmount: string; exchangeAmount: string }) => {
```

Comment:
Let's define this:

```typescript
type PartialAmounts = Pick<ExchangedFlipInputAmounts, 'nativeAmount' | 'exchangeAmount'>
```

And then:

```typescript
onAmountsChanged = (amounts: PartialAmounts) => {
  const { exchangeAmount, nativeAmount } = amounts
  ...
}
```

This connects our `onAmountsChanged` implementation to the `FlipInputModal2` type definitions.